pandas/core/frame.py

```diff
@@ -183,7 +183,8 @@
     the order of the join keys depends on the join type (how keyword).
 suffixes : 2-length sequence (tuple, list, ...)
     Suffix to apply to overlapping column names in the left and right
-    side, respectively.
+    side, respectively. If (False, False), overlapping
+    column names raise an error.
```

Comment:
can you add an example in the doc-string to make this even more clear? (we don't even have a suffix example at all......)

Reply:
@jreback, the doc string example does illustrate the behavior for the default suffix arguments.

Also, there is a more detailed suffix example here: https://pandas.pydata.org/pandas-docs/stable/merging.html#overlapping-value-columns

I suggest adding a specific note or example to the link above for `suffix=(False, False)` instead of the doc string.  

I feel a more experienced user would likely know what to do just from the parameter argument description while a more novice user will probably need to look at the longer documentation with copy/paste examples that work with ipython or jupyter.    

I also suggest the merge, join, and concat methods all have a reference to the more complete documentation in the *see also* section.  I've found this documentation very helpful in the past and still refer to if from time to time.  

Thoughts?

Reply:
@jreback Also, I'm happy to just update the doc string.  Just wanted to offer a different perspective.  Let me know what you prefer.  Thanks!

Reply:
@elmq0022 yeah an updated doc-string would be great

Reply:
For the type, I'd have `suffixes : tuple of (str, str), default ('_x', '_y')`. Small detail, but I'd rephrase you addition to sound like `To raise an exception on overlapping columns, use (False, False)`. Being able to use (False, False) to raise looks like a feature to me, and I think it's better to present it this way. May be we could also have it in the `Notes` section.

But it's just an idea, I'm happy with the PR the way it is. It's a good addition.